Install.md

```diff
@@ -0,0 +1,86 @@
+# Anomaly Detection Installation
+This document describes how to set up Anomaly Detection for OpenSDS project.
+
+## Pip Installation
+Run command ```which pip``` to check if your environment has pip tool, if not you can refer to the following steps to install it.
+```shell
+$ wget https://bootstrap.pypa.io/get-pip.py
+$ python get-pip.py # Maybe python3 get-pip.py in your environment.
+``` 
+
+## Anomaly Detection Installation
+Install anomaly detection using pip.
+```shell
+pip install git+https://github.com/opensds/anomaly-detection.git
+```
+if you want to install a specified branch, run command like blow:
+```shell
+$ pip install git+https://github.com/opensds/anomaly-detection.git@0.5.3
+```
+## Configuration
+Anomaly detection will get metrics from from Telemetry, before modifying the configuration file, please make sure the telemetry is started up.
```

Comment:
Do we have documents on how to start up telemetry?  Please point to the documents if they exist, otherwise add a note that document for telemetry will be added later.